university/src/university/University.java

```diff
@@ -0,0 +1,62 @@
+package university;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class University {
+    private static University UNIVERSITYINSTANCE;
+    private static Set<Course> courseSet;
+    private static Set<Professor> professorSet;
+    private static Set<Enrollable> participantSet;
+
+    University() {
+        courseSet = new HashSet<>();
+        professorSet = new HashSet<>();
+        participantSet = new HashSet<>();
+    }
+
+    public static University getUniversityInstance() {
+        if (UNIVERSITYINSTANCE == null) {
+            UNIVERSITYINSTANCE = new University();
+        }
+        return UNIVERSITYINSTANCE;
+    }
+
+    public void offerCourse(Course course) {
+        if (courseSet.contains(course)) {
+            System.out.println("This course is already exist.");
```

Comment:
Please note that `exist` is a verb - thus, instead of `is exist`, you should use the present simple conjugation: `This course already exists.` Alternatively, you could say `This course is already existent.`